bedrock/firefox/templates/firefox/android/index.html

```diff
@@ -311,36 +311,28 @@ <h2 class="heading">{{ _('Keep your private info private') }}</h2>
     </p>
 
     <ul>
-      <li id="dnt">
-        <h3>{{ _('Do not track') }}</h3>
-
-        <p>
-        {% trans %}
-          Enable this Firefox innovation to help you stay in control of how your browsing
-          information is collected and used online.
-        {% endtrans %}
-        </p>
-
-        <a href="{{ url('firefox.dnt') }}">{{ _('Learn more about Do Not Track') }}</a>
-      </li>
-      <li id="private-browsing">
-        <h3>{{ _('Private Browsing') }}</h3>
-
-        <p>
-        {% trans %}
-          When activated, Firefox for Android won’t accept cookies, remember passwords or
-          record any sites that you’ve visited.
-        {% endtrans %}
-        </p>
-
-        <a rel="external" href="https://www.youtube.com/watch?v=Ewk0YtFTBec&amp;list=PLA8ABF6B7A66A0C5E&amp;index=15">
-          {% if l10n_has_tag('android_spring_updates') %}
-            {{ _('Learn how to open a private window') }}
-          {% else %}
-            {{ _('See how Private Browsing works') }}
-          {% endif %}
-        </a>
-      </li>
+    {% if l10n_has_tag('tracking_protection') %}
+        <li id="private-browsing">
+          <h3>{{ _('Private Browsing with<br /> Tracking Protection') }}</h3>
+
+          <p>
+            {{ _('Browse without having your history, searches or cookies saved, and block hidden third parties from following you across the Web.') }}
+          </p>
+
+          <a class="more" rel="external" data-interaction="outbound link" href="https://support.mozilla.org/kb/private-browsing-browse-web-without-saving-info">{{ _('Learn more about Private Browsing') }}</a>
+        </li>
+      {% else %}
+        <li id="private-browsing">
+          <h3>{{ _('Private Browsing') }}</h3>
+
+          <p>
+            {{ _('Sometimes it’s nice to go undercover.') }}
```

Comment:
Would this line and the one below be better as a single string instead of split into two? Thoughts @flodolo?